safenode/src/bin/kadnode.rs

```diff
@@ -54,11 +62,13 @@ async fn main() -> Result<()> {
             match event {
                 NetworkEvent::RequestReceived { req, channel } => {
                     // Reply with the content of the file on incoming requests.
-                    if let Request::Query(Query::GetChunk(xor_name)) = req {
-                        let addr = ChunkAddress(xor_name);
-                        let chunk = storage_clone.query(&addr).await.unwrap();
+                    if let Request::Query(Query::GetChunk(addr)) = req {
+                        let chunk = storage_clone.query_chunk(&addr).await.unwrap();
```

Comment:
`storage_clone.query_chunk(&addr)`
I think this is more consistent and suitable: `get_chunk`, as that is the specific name of the query (as in `GetChunk`, as is the name of that query variant).